mirage/psf/psf_library.py

```diff
@@ -1,120 +1,12 @@
-import os
+import astropy.convolution
 import itertools
-
-import webbpsf
 import numpy as np
-import astropy.convolution
+import os
+import webbpsf
 from astropy.io import fits
```

Comment:
A nitpicky thing since you're implementing PEP8 changes here: I believe the official PEP8 style is to separate imports by category (standard library, third party, then local) and then order alphabetically within that. So this would rather look like:
```
import itertools
import os

import astropy.convolution
from astropy.io import fits
import numpy as np
import webbpsf
```